### DISCUSSION TITLE
How to install blockscout local using docker compose integration

### DISCUSSION THREAD

**Discussion by PazhaniMani** [blockscout/blockscout#8781] [Support]
### Description

I tried your documentation for docker integration and put this command 
cd ./docker-compose
docker-compose up --build
but loading 2 hours after error below
![WhatsApp Image 2023-11-01 at 6 29 22 PM](https://github.com/blockscout/blockscout/assets/69962094/9154dac2-a7b0-4e52-954e-974ad1011ea6)

### Type of the installation

Docker-compose

### Type of the JSON RPC archive node

Geth

### Type of the chain

L1

### Steps to reproduce

![Screenshot at Nov 02 09-32-24](https://github.com/blockscout/blockscout/assets/69962094/38e774da-e8e0-4eb8-a6ba-627d339f5b7a)
I can follow this step, but error after 2 hours loading..... below like this

![WhatsApp Image 2023-11-01 at 6 29 22 PM](https://github.com/blockscout/blockscout/assets/69962094/755842a7-b206-46c9-93a5-1b7700e655aa)

so how to fix it or any other installation procedure, because I Am a new learner of blockscout

### Backend version

v5.3.1

### Frontend version

v1.11.1

### Elixir & Erlang/OTP versions

Elixir 1.15.7 (compiled with Erlang/OTP 26)

### Operating system

macOS

**Comment by sl1depengwyn** [comment]
Duplicate of https://github.com/blockscout/blockscout/issues/8778
